src/components/ChooseCurrency.tsx

```diff
@@ -1,119 +1,37 @@
-    SelectField,
-    VStack
-} from "~/components";
+import { Button, ExternalLink, InfoBox, NiceP, VStack } from "~/components";
 import { useI18n } from "~/i18n/context";
 import { useMegaStore } from "~/state/megaStore";
-import { eify, timeout } from "~/utils";
-
-export interface Currency {
-    value: string;
-    label: string;
-    hasSymbol?: string;
-    maxFractionalDigits: number;
-}
+import {
+    BTC_OPTION,
+    Currency,
+    eify,
+    FIAT_OPTIONS,
+    timeout,
+    USD_OPTION
+} from "~/utils";
 
 type ChooseCurrencyForm = {
     fiatCurrency: string;
 };
 
-/**
- *  FIAT_OPTIONS is an array of possible currencies
- *  All available currencies can be found here https://api.coingecko.com/api/v3/simple/supported_vs_currencies
- *  @Currency
- *  @param {string} label - should be in the format {Name} {ISO code}
- *  @param {string} values - are uppercase ISO 4217 currency code
- *  @param {string?} hasSymbol - if the currency has a symbol it should be represented as a string
- *  @param {number} maxFractionalDigits - the standard fractional units used by the currency should be set with maxFractionalDigits
- *
- *  Bitcoin is represented as:
- *  {
- *      label: "bitcoin BTC",
- *      value: "BTC",
- *      hasSymbol: "₿",
- *      maxFractionalDigits: 8
- *  }
- */
-
-export const FIAT_OPTIONS: Currency[] = [
-    {
-        label: "Bitcoin BTC",
-        value: "BTC",
-        hasSymbol: "₿",
-        maxFractionalDigits: 8
-    },
-    {
-        label: "United States Dollar USD",
-        value: "USD",
-        hasSymbol: "$",
-        maxFractionalDigits: 2
-    },
-    { label: "Swiss Franc CHF", value: "CHF", maxFractionalDigits: 2 },
-    {
-        label: "Chinese Yuan CNY",
-        value: "CNY",
-        hasSymbol: "¥",
-        maxFractionalDigits: 2
-    },
-    {
-        label: "Euro EUR",
-        value: "EUR",
-        hasSymbol: "€",
-        maxFractionalDigits: 2
-    },
-    {
-        label: "Brazilian Real BRL",
-        value: "BRL",
-        hasSymbol: "R$",
-        maxFractionalDigits: 2
-    },
-    {
-        label: "British Pound GBP",
-        value: "GBP",
-        hasSymbol: "₤",
-        maxFractionalDigits: 2
-    },
-    {
-        label: "Australia Dollar AUD",
-        value: "AUD",
-        hasSymbol: "$",
-        maxFractionalDigits: 2
-    },
-    {
-        label: "Japanese Yen JPY",
-        value: "JPY",
-        hasSymbol: "¥",
-        maxFractionalDigits: 0
-    },
-    {
-        label: "Korean Won KRW",
-        value: "KRW",
-        hasSymbol: "₩",
-        maxFractionalDigits: 0
-    },
-    { label: "Kuwaiti Dinar KWD", value: "KWD", maxFractionalDigits: 3 }
-].sort((a, b) => (a.value > b.value ? 1 : b.value > a.value ? -1 : 0));
-
-export const USD_INDEX = FIAT_OPTIONS.findIndex((fo) => fo.value === "USD");
-export const BTC_INDEX = FIAT_OPTIONS.findIndex((fo) => fo.value === "BTC");
+const COMBINED_OPTIONS: Currency[] = [USD_OPTION, BTC_OPTION, ...FIAT_OPTIONS];
```

Comment:
I was thinking of putting `USD_OPTION, BTC_OPTION` inside of `FIAT_OPTIONS` but this works too